demos/crossroad_camera_demo/README.md

```diff
@@ -8,7 +8,7 @@ reports person attributes like gender, has hat, has long-sleeved clothes
 * `person-reidentification-retail-0079`, which is executed on top of the results from the first network and prints
 a vector of features for each detected person. This vector is used to conclude if it is already detected person or not.
 
-For more information about the pre-trained models, refer to the [model documentation](../../intel_models/index.md).
+For more information about the pre-trained models, refer to the [models documentation](../../intel_models/index.md).
```

Comment:
This change is not in the internal repository, and it's also questionable WRT grammar, so please revert it.

Reply:
Fixed